02-express-tutorial/videoPractice.js

```diff
@@ -0,0 +1,65 @@
+const http = require('http')//******problem with ()and ''*****
```

Comment:
Try not to commit files that aren't part of the homework assignment.  This makes things more confusing and in the real world it's better to keep things as concise as possible!